lib/new_relic/agent/monitors/synthetics_monitor.rb

```diff
@@ -16,6 +17,7 @@ def on_finished_configuring(events)
 
       def on_before_call(request) # THREAD_LOCAL_ACCESS
         encoded_header = request[SYNTHETICS_HEADER_KEY]
+        info_header = request[SYNTHETICS_INFO_HEADER_KEY]
         return unless encoded_header
```

Comment:
2 things here...

- Naming-wise, now looking at "encoded_header" and "info_header", I'm tempted to consider "info_header" as non-encoded. Given that the new header is an "info" type header, is there anything that be used to help with naming of the original header? Perhaps "base_header" or something?
- The early `return` will continue to happen unless the original header is present. Is it possible to have the new header present without the original header?

Reply:
> Naming-wise, now looking at "encoded_header" and "info_header", I'm tempted to consider "info_header" as non-encoded. Given that the new header is an "info" type header, is there anything that be used to help with naming of the original header? Perhaps "base_header" or something?
You're correct that info_header is not encoded, it is just straight json. I didn't originally change the name because of that, but if you think base header is better I can totally do that.


> The early return will continue to happen unless the original header is present. Is it possible to have the new header present without the original header?
No, this isn't possible, the new header is just additional information and can't be used on its own.

Reply:
Excellent, thank you. No actions needed here then.